### PR TITLE
add 2 music channels, add 2 ids

### PR DESCRIPTION
**Pull request by Owyn**
any idea why it gives `offline` for all these and not `blocked`? cuz it plays fine for me here

### CODE REVIEW

File: streams/ru.m3u [hunk 1]
```diff
@@ -768,7 +768,11 @@ https://sc.id-tv.kz/1KanalEvraziya.m3u8
...
 #EXTINF:-1 tvg-id="" status="error",Телестанция МИР (Новосибирск) (720p) [Not 24/7]
 http://serv30.vintera.tv:8081/stanciya_mir/mir/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",RU.TV (1080p)
+#EXTINF:-1 tvg-id="RuTV.ru" status="error",RU.TV (1080p)  [Geo-blocked]
 https://strm.yandex.ru/kal/rutv_cv/rutv_cv0.m3u8
-#EXTINF:-1 tvg-id="" status="error",Ю (576p)
+#EXTINF:-1 tvg-id="U.ru" status="error",Ю (576p) [Geo-blocked]
 https://strm.yandex.ru/kal/utv/utv0.m3u8
+#EXTINF:-1 tvg-id="EuropaPlusTV.ru" status="",Europa Plus TV (1080p) [Geo-blocked]
+http://strm.yandex.ru/kal/europaplus/europaplus0_169_1080p.json/index-v1-a1.m3u8
```
Comment (LaneSh4d0w) [inline streams/ru.m3u:776]:
```suggestion
http://strm.yandex.ru/kal/europaplus/europaplus0.m3u8
```

File: streams/ru.m3u [hunk 2]
```diff
@@ -768,7 +768,11 @@ https://sc.id-tv.kz/1KanalEvraziya.m3u8
...
+#EXTINF:-1 tvg-id="TNTMusic.ru" status="",ТНТ Music (1080p) [Geo-blocked]
+http://strm.yandex.ru/kal/tntmusic/tntmusic0_169_1080p.json/index-v1-a1.m3u8
```
Comment (LaneSh4d0w) [inline streams/ru.m3u:778]:
```suggestion
http://strm.yandex.ru/kal/tntmusic/tntmusic0.m3u8
```

**Comment by LaneSh4d0w** [comment]
GH bot services are outside Russia, and it basically tries to see what answer the server gives. Server gives a 451 Not Available for Legal Reasons error code, which is not something the bot understands (it doesn't understand it as a geo-block, think about 403 in most servers), hence why it puts it as Offline.

**Comment by Owyn** [comment]
Thx for changes to auto-quality channels.... 
but now check fails for some apparent reason? even after updating and resolving conflict

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**